### PR TITLE
docs: add rtl message/notification explanation

### PR DESCRIPTION
**Pull request by xrkffgg**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution
- 添加 message / notification RTL 说明

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           none|
| 🇨🇳 Chinese |    none        |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/message/index.en-US.md](https://github.com/ant-design/ant-design/blob/docs-rtl-message/notification/components/message/index.en-US.md)
[View rendered components/message/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/docs-rtl-message/notification/components/message/index.zh-CN.md)
[View rendered components/notification/index.en-US.md](https://github.com/ant-design/ant-design/blob/docs-rtl-message/notification/components/notification/index.en-US.md)
[View rendered components/notification/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/docs-rtl-message/notification/components/notification/index.zh-CN.md)

### CODE REVIEW

File: components/notification/index.zh-CN.md [hunk 1]
```diff
@@ -50,6 +50,9 @@ config 参数如下：
 还提供了一个全局配置方法，在调用前提前配置，全局一次生效。
 
 - `notification.config(options)`
+  > 当你使用 `ConfigProvider` 进行全局化配置时，系统会默认自动开启 RTL 模式。
```
Comment (afc163) [inline components/notification/index.zh-CN.md:53]: 上面加个空行。

File: components/message/index.en-US.md [hunk 2]
```diff
@@ -65,6 +65,10 @@ Methods for global configuration and destruction are also provided:
 
 #### message.config
 
+> When you use `ConfigProvider` for global configuration, the system will automatically start RTL mode by default.
+>
+> When you want to use it alone, you can start the RTL mode through the following settings.
```
Comment (afc163) [inline components/message/index.en-US.md:70]: 缩进应该还是可以加上，这样不破坏列表样式。

Comment (xrkffgg) [reply components/message/index.en-US.md:70]:
```suggestion
  > When you want to use it alone, you can start the RTL mode through the following settings.
```

你是说这样吗？
这 2 个在 样式上是一样的

Comment (xrkffgg) [reply components/message/index.en-US.md:70]: message 使用了 ###，两种是一样的

notification 使用了 -  ，是有缩进

**Review by afc163 — APPROVED** [review]

**Pull request merged**